src/odemis/gui/cont/acquisition.py

```diff
@@ -1397,9 +1397,11 @@ def __init__(self, tab_data, tab_panel, projectbar_ctrl, calibration_ctrl):
 
         # Path to the acquisition
         # <External storage>/<date>/<project name>/<roa name>
-        # TODO: display proper path to external storage (if mounted)
         self.path = datetime.today().strftime('%Y-%m-%d')
-        self._tab_panel.txt_destination.SetValue("<External Storage>/%s" % self.path)
+        if self._main_data_model.asm:
+            self._main_data_model.asm.externalStorageURL.subscribe(self._on_storage_dir, init=True)
+        else:
```

Comment:
Do we still need the else statement for any case? We always have the ASM component in all backends and simulators right? And I think it is not possible to start the backend without specifying the directory on the external storage (you could do a quick try by commenting it out) as it is also read in the technolution driver for the .externalStorageURL VA.

Reply:
I was mostly concerned with the simulators, especially the ones in this repository under usr/bin/share/odemis/sim. We don't want to start the external ASM simulator here. But I agree that it's a bit annoying to have a special case just for this. How about I just add a comment that normally, there should be an ASM and the corner case is for the simulator?

Reply:
True, I forgot about this file. It is the only one without an ASM simulator. Yes, maybe a comment is a good solution for now. I don't have a better idea for now.